src/sema/expression.rs

```diff
@@ -7342,19 +7488,19 @@ pub fn named_function_call_expr(
 }
 
 /// Get the return values for a function call
-fn function_returns(ftype: &Function) -> Vec<Type> {
-    if !ftype.returns.is_empty() {
+fn function_returns(ftype: &Function, resolve_to: &ResolveTo) -> Vec<Type> {
```

Comment:
@seanyoung This is where I change the return type of the function. When I checked the CFG everything was correct, however the problem of `ResolveTo::Discard` arose when I set `Type::Void` to single return functions.